reporter/html.js

```diff
@@ -453,10 +453,14 @@ function storeFixture() {
 }
 
 function appendUserAgent() {
-	var userAgent = id( "qunit-userAgent" );
+	var userAgent = id( "qunit-userAgent" ),
+		versionInfo = "QUnit " + QUnit.version;
+
 	if ( userAgent ) {
 		userAgent.innerHTML = "";
-		userAgent.appendChild( document.createTextNode( navigator.userAgent ) );
+		userAgent.appendChild(
+			document.createTextNode( navigator.userAgent + ": " + versionInfo )
```

Comment:
Let's replace the colon with a semicolon. Could also inline the `versionInfo` variable.